content/english/java-tictactoe/answer-key.md

```diff
@@ -44,17 +47,21 @@ public class TicTacToe_Nuevo {
         while(true){
             int move;
 			try {
+                // get integer input
```

Comment:
It seems the comments don't line up exactly in Github because of spaces vs. tabs, can we make this consistent?